### PR TITLE
Update documentation on how to manually install HDM.

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: MANUAL_INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Manual install
+
+## General
+
+HDM binds per default to port 3000.
+
+## Ruby
+
+Make sure you have the necessary ruby version on your system avaiable. You might use rvm or rbenv.
```
Comment (bastelfreak) [inline MANUAL_INSTALL.md:9]: maybe add a note that the minimal supported version is in .ruby-version file?

Comment (rwaffen) [reply MANUAL_INSTALL.md:9]: added.

File: MANUAL_INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+## CentOS 8 Streams
+
+### **RVM**
+
+As root do:
+
+    command curl -sSL https://rvm.io/mpapis.asc | sudo gpg2 --import -
```
[the remark this answers is not in the document]
Comment (rwaffen) [reply MANUAL_INSTALL.md]: updated.

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -117,6 +72,4 @@ Password: puppetlabs
 
 HDM: `http://puppet.pe.psick.io:3000`
```
Comment (tuxmea) [inline DEVELOPMENT.md:73]: Please remove the mentioning of psick.

**Review by SimonHoenscheid — COMMENTED** [review]
LGTM, we just keep the RVM section for reference instead of writing: HDM requires the Ruby versin mentioned in .ruby-version to be available on the system?

**Comment by rwaffen** [comment]
i just typed down a complete installation. this included rvm. so i can make sure, this manual installation really works.

**Pull request merged**